src/Form/MediaLibraryEmbeddableForm.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Drupal\stanford_media\Form;
+
+use Drupal\Core\Form\FormBuilderInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\media_library\Form\OEmbedForm;
+
+/**
+ * Media library add stanford embed input form.
+ *
+ * @package Drupal\stanford_media\Form
+ */
+class MediaLibraryEmbeddableForm extends OEmbedForm {
+
+  /**
+   * {@inheritDoc}
+   */
+  public function getFormId() {
+    return $this->getBaseFormId() . '_embeddable';
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function buildInputElement(array $form, FormStateInterface $form_state) {
+    // This was adapted from \Drupal\media_library\Form\OembedForm.
+    $user = \Drupal::currentUser();
+    $authorized_for_unstructured = FALSE;
+    if ($user->hasPermission('create field_media_embeddable_code') ||
+        $user->hasPermission('edit field_media_embeddable_code')) {
+      $authorized_for_unstructured = TRUE;
+    }
+
+    $media_type = $this->getMediaType($form_state);
+    $providers = $media_type->getSource()->getProviders();
+
+    $form['container'] = [
+      '#type' => 'container',
+    ];
+
+    $form['container']['field_media_embeddable_oembed'] = [
+      '#type' => 'url',
+      '#title' => $this->t('Add @type via URL', [
+        '@type' => $this->getMediaType($form_state)->label(),
+      ]),
+      '#description' => $this->t('Allowed providers: @providers. For custom embeds, please request support.', [
+        '@providers' => implode(', ', $providers),
+      ]),
+      '#required' => FALSE,
```

Comment:
`false` is the default value. this line can be removed unless the field is actually required.